### ISSUE TITLE
Issue on page /index.html

### ISSUE THREAD

**Issue by TaMcG412** [dorisavedikian/NALGuideToSubjectIndexing#3]
Update Table of Contents - This is the latest of several introductory sections I created. It includes the table of contents as well as the first few sections. There are two comments in this section. As I mentioned yesterday, I used the comments option to add citations so that the text would not become clogged with citation metadata. I think you mentioned adding links so that they are not cluttering up the page. I think that was part of the problem with some previous versions of the indexing guidelines. There were so many examples that it was sometimes hard to follow the discussion. Some examples were also taken out of context so they could be misleading if someone tried to search for them. 

I broke out the rest of the document into individual sections so that it would be easier for everyone to read and edit. I'm providing this one first since it will not change significantly in the near term. I'll upload some more next week so you can get the feel for what I have been doing. That said, there will be a lot more editing once the other indexers start providing their changes. 

If you have any questions, feel free to email me (teresa.mcgervey@usda.gov) 

[1_Draft Introductory sections_03March2023.docx](https://github.com/dorisavedikian/NALGuideToSubjectIndexing/files/10883142/1_Draft.Introductory.sections_03March2023.docx)

**Comment by dorisavedikian** [comment]
@TaMcG412 Thanks Teresa! I updated the guide with the content you gave me. Please check it out and let me know if you see anything you would like to change.

**Comment by dorisavedikian** [comment]
This is complete.
